tests/small-build/test_api.py

```diff
@@ -0,0 +1,18 @@
+from conceptnet5 import api
+from nose.tools import eq_
+
+
+def test_related_query():
+    # Test that we can look up related terms
+    result = api.query_related('/c/en/test', limit=3)
+    print(result)
```

Comment:
Do those print statements serve a purpose or are just leftovers from testing the tests?